sample/www/jxcore/thali/mockmobile.js

```diff
@@ -0,0 +1,58 @@
+var keys = {};
+
+function Mobile(key) {
+  keys[key] || (keys[key] = new NativeCall(key));
+  return keys[key];
+}
+
+Mobile.invokeNative = function (key, arg) {
+  keys[key].registerNativeCallback(arg);
+};
+
+function invokeCallback(key, args) {
+  var localArgs = keys[key].callNativeArguments;
+  var cb = localArgs[localArgs.length - 1];
+  cb.apply(null, args);
+}
+
+Mobile.invokeConnect = function () {
+  var len = arguments.length, args = new Array(len);
```

Comment:
Since the next four methods are all identical in contents but for a string why not just refactor this into a call to a single function?